apps/www/_blog/2022-08-15-launch-week-five-cli-v1-and-admin-api-beta.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: 'Supabase CLI v1 and API beta'
+description: We are moving Supabase CLI v1 out of beta, and releasing Admin API beta.
+author: soedirgo,qiao
+image: lw5-cli/thumbnail.jpg
+thumb: lw5-cli/thumbnail.jpg
+tags:
+  - launch-week
+date: '2022-08-15'
+toc_depth: 3
+---
+
+
+Today we are moving the Supabase CLI v1 out of beta. The Supabase CLI is capable of managing database migrations and generating TypeScript types. Follow these [install instructions](https://github.com/supabase/cli#getting-started) to get started.
+
+In addition, we are also announcing the Admin API beta. The Admin API is a REST API that allows you to manage organizations, projects, Edge Functions, and more. You can read the [API docs](https://supabase.com/docs/reference/api) or interact with the Admin API from the Supabase CLI v1.
+
+## Manage organizations, projects, Edge Functions, and more
+
+If you've used [Supabase Edge Functions](https://supabase.com/blog/2022/03/31/supabase-edge-functions#quickstart), you've used the Supabase CLI. We're extending it to let you manage organizations and projects.
+
+```bash
+supabase login
+supabase orgs list
+supabase projects create my-project --org-id cool-green-pqdr0qc --db-password ******** --region us-east-1
+```
+
+Once your new Supabase project is created, use the CLI to link it locally to begin development.
+
+```bash
+supabase link --project-ref <project-id>
+```
+
+You can also [call the Admin API endpoint directly](#admin-api) to create a project.
```

Comment:
```suggestion
```

Remove this in my opinion, it's becoming a distraction to keep mentioning the Admin API